jcloud/flow.py

```diff
@@ -69,6 +72,12 @@ def _exit_error(text: str):
     exit(1)
 
 
+def get_resource_url(resource: str) -> str:
+    if resource == Resources.Job:
+        return JOBS_API
+    return SECRETS_API
```

Comment:
same as above, if the resouces is invalid, we should not progress. Only progress the type we have.

Reply:
same here the parser actually carries out this check before